cpp/src/io/parquet/page_enc.cu

```diff
@@ -3137,6 +3429,13 @@ void EncodePages(device_span<EncPage> pages,
     gpuEncodeDeltaLengthByteArrayPages<encode_block_size>
       <<<num_pages, encode_block_size, 0, strm.value()>>>(pages, comp_in, comp_out, comp_results);
   }
+  if (BitAnd(kernel_mask, encode_kernel_mask::DELTA_BYTE_ARRAY) != 0) {
```

Comment:
I've seen this utility in multiple places and just realized it's just `cuda::std::bit_and()` defined in `<cuda/std/functional>`.

could be a separate PR removing this also `BitOr`

Reply:
There's a slight difference in that `BitAnd` allows either argument to be a scoped enum. Since the scoped enums can't be implicitly cast to an int, I don't think the `std::bit_and()` will be happy. I was trying to avoid `static_cast<uint32_t>` all over the place, and @vuule _really_ didn't want me overloading 'operator &()` :joy:. But we can revisit this.

Reply:
Oh my... ignore me then. I'm happy with the current the way in this case.